platform/source/mbed_power_mgmt.c

```diff
@@ -28,6 +28,12 @@
 #include "platform/mbed_wait_api.h"
 
 #include <stdio.h>
+#ifdef MBED_SLEEP_TRACING_ENABLED
+#include <string.h>
+#endif
+
+/* Uncomment next line for STM32 with LPTICKER based on LPTIM */
+// #define STM32_SLEEP_TRACING
```

Comment:
Can this be a config rather than leaving this here?

Reply:
Rather rely on someone defining `-D` than touching this file and changing it for this.